wiris/html-integrations
# KB-2971: Fix CKEditor5 not rendering LaTeX when updating $$latex$$ to an image

## Description

This pull request fixes an issue with the CKEditor 5 plugin.

When calling `editor.getData()` (or when clicking the "Update" button in the HTML5 demos), any formulas written out as LaTeX expressions (e.g. "$$x^2$$") are not rendered as MathML.

## How to reproduce

1. Open a CKEditor 5 HTML5 demo
2. Write some text with a LaTeX expression, e.g. "$$x^2$$"
3. Click "Update"

Expected: the LaTeX is rendered as a formula image. 
Actual: the LaTeX stays as is.
